examples/ota-requestor-app/linux/main.cpp

```diff
@@ -110,6 +115,9 @@ bool HandleOptions(const char * aProgram, OptionSet * aOptions, int aIdentifier,
     case kOptionRequestorCanConsent:
         gRequestorCanConsent.SetValue(true);
         break;
+    case kOptionOtaDownloadPath:
+        chip::Platform::CopyString(gOtaDownloadPath, aValue);
```

Comment:
How do we know aValue fits in gOtaDownloadPath?

Reply:
CopyString infers the array size- i.e. the compiler deduces it and generates the corresponding templatized function code.
CopyString's function header:
```
template <size_t N>
inline void CopyString(char (&dest)[N], const char * source)
```